data.tf

```diff
@@ -66,3 +66,63 @@ data "aws_iam_policy_document" "update_lambda_edge" {
     }
   }
 }
+
+data "aws_iam_policy_document" "sign_code" {
+  statement {
+    sid = "UploadToS3"
+
+    actions = [
+      "s3:PutObjectTagging",
+      "s3:PutObjectAcl",
+      "s3:PutObject",
+      "s3:GetObjectVersion",
+      "s3:GetObjectTagging",
+      "s3:GetObject",
+      "s3:DeleteObject",
+    ]
+
+    resources = [
+      "arn:aws:s3:::${var.signing_bucket_name}/*",
+    ]
+  }
+
+  statement {
+    sid = "ListBucket"
+
+    actions = [
+      "s3:ListBucket",
+      "s3:ListBucketVersions",
+    ]
+
+    resources = [
+      "arn:aws:s3:::${var.signing_bucket_name}",
+    ]
+  }
+
+  statement {
+    sid = "ListDescribeSigningJob"
+
+    actions = [
+      "signer:DescribeSigningJob",
+      "signer:ListSigningJobs",
+      "signer:ListSigningProfiles",
+    ]
+
+    resources = [
+      "*",
```

Comment:
necessary to give `*` ? pls add checkov skip.

Reply:
Yes. These actions only support all the resources wildcard "*". I've just added the checkov skip comment.